openshift-kube-apiserver/admission/customresourcevalidation/node/validate_node_config.go

```diff
@@ -57,7 +78,11 @@ func toConfigNodeV1(uncastObj runtime.Object) (*configv1.Node, field.ErrorList)
 	return obj, nil
 }
 
-type configNodeV1 struct{}
+type configNodeV1 struct {
```

Comment:
If you could embed `admission.Handler` into the `configNodeV1`, then we could use `SetReadyFunc` and `WaitForReady` directly:
```
type configNodeV1 struct {
	*admission.Handler
	...
}
```

Reply:
ooh fancy

Reply:
updated